src/server/apollo/typeDefs.ts

```diff
@@ -62,12 +62,25 @@ export const typeDefs = gql`
     talks(day: String): [Talk!]!
   }
 
+  type Poster {
+    id: ID!
+    title: String!
+    abstract: String
+    description: String
+    day: String!
+    type: TalkType!
+    pdflink: String
+    speakers: [Speaker!]!
+
+  }
+
   enum TalkType {
     BIRDS_OF_FEATHER
     BREAK
     KEYNOTE
     LIGHTNING_TALK
     MINISYMPOSIUM
+    POSTER
```

Comment:
We're doing posters as a different GraphQL type instead, so this enum value isn't needed.

The alternative might be to not create the `Poster` type in GraphQL and add this enum value, but since we _are_ creating the `Poster` GraphQL type, we don't need this.
```suggestion
```